common/djangoapps/util/views.py

```diff
@@ -249,10 +250,17 @@ def _record_feedback_in_zendesk(
     """
     zendesk_api = _ZendeskApi()
 
-    additional_info_string = (
-        u"Additional information:\n\n" +
-        u"\n".join(u"%s: %s" % (key, value) for (key, value) in additional_info.items() if value is not None)
-    )
+    # Extract information that can be stored in custom fields from the additional_info dictionary.
+    zendesk_custom_fields = []
+    zendesk_additional_info = []
+    for key, value in sorted(additional_info.items()):
+        if value is None:
+            continue
+        elif key in settings.ZENDESK_CUSTOM_FIELDS:
+            zendesk_custom_fields.append({"id": settings.ZENDESK_CUSTOM_FIELDS[key], "value": value})
+        else:
+            zendesk_additional_info.append("{}: {}".format(key, value))
+    zendesk_additional_info = u"Additional information:\n\n" + u"\n".join(zendesk_additional_info)
```

Comment:
This is the string that actually gets sent to zendesk, built from the dictionary of additional_info passed to this method.